### ISSUE TITLE
Code is incomplete

### ISSUE THREAD

**Issue by JattJames** [nitishpandey/semiprecious#2]
H, was looking over, your code is incomplete and there is no Database attached, also it is only showing 1000 files as per github, other files not included

**Comment by nitishpandey** [comment]
> H, was looking over, your code is incomplete and there is no Database attached, also it is only showing 1000 files as per github, other files not included

True. We couldn't get the full code up here. Will try againg. How to attach the database though?

**Comment by JattJames** [comment]
For Database, you can try zipping it and Upload it or use any external link, 

as far as files are concerned, i think better to zip and upload it

**Comment by nitishpandey** [comment]
Thanks.
Didn't know we could zip them and upload
